DEVELOPMENT.md

```diff
@@ -74,7 +75,7 @@ Export interfaces for exported functions/components. Typescript will generate de
 
   Don't use ~~!important~~, if really needed - for specificity hack you can use `.fi-button.button--custom.button--custom {...}`
 
-- TBD: CSS-exports
+- TBD: CSS-exports (tested to be working , not implemented for all components)
```

Comment:
TBD or TODO? Should we capture this in a separate issue instead of adding to Dev MD?

Reply:
It is not decided that it will be done. Not done if not used. But having a test that it can be done and not creating changes that would not allow it - we are answering that CSS-in-JS is not a must requirement for everything.